pkg/ui/workspaces/db-console/src/redux/apiReducers.ts

```diff
@@ -77,7 +78,15 @@ function rollupStoreMetrics(
 
 export const nodesReducerObj = new CachedDataReducer(
   (req: api.NodesRequestMessage, timeout?: moment.Duration) =>
-    api.getNodesUI(req, timeout).then(rollupStoreMetrics),
+    api
+      .getNodesUI(req, timeout)
+      .then(rollupStoreMetrics)
+      .then(nodeStatuses => {
+        return nodeStatuses.map(ns => {
```

Comment:
nit: should this be a `forEach` instead of a `map`?

Reply:
Done